cli/flox/src/utils/tracing.rs

```diff
@@ -0,0 +1,20 @@
+use sentry::configure_scope;
+
+/// Sets a tracing tag for the current scope. These always get rolled up to the
+/// root transaction/span but that shouldn't make a difference for searching
+/// and we don't use any child spans at the time of writing anyway.
```

Comment:
i think we _do_ use child spans:

* https://github.com/flox/flox/blob/623161b1ebf2c91e1d4ead49c8f4a21fb3f2161a/cli/flox/src/commands/edit.rs#L101-L102

* https://github.com/flox/flox/blob/623161b1ebf2c91e1d4ead49c8f4a21fb3f2161a/cli/flox/src/commands/update.rs#L50-L51

 and a few more.
 
 I guess if they get rolled up thats good enough, ~can you check if they are namespaced in that case?~
 
 apparently they arent namespaced, oh well.. ^^

Reply:
I also wrote this before I realised that we have an extra span in `commands::services` that I showed in the demo. I'll update the comment accordingly. I think it's still fine though; in other tracing systems that I've used in the past we ended up favouring really wide and shallow traces because it made it easier to search and less cost prohibitive.